src/ts/SidePanel.class.ts

```diff
@@ -38,7 +38,7 @@ export default class SidePanel {
     // show panel
     document.getElementById(this.panelId).removeAttribute('hidden');
     // store the current state
-    localStorage.setItem(`is${this.model}Open`, '1');
+    localStorage.setItem('opened-sidepanel', this.model);
```

Comment:
```suggestion
    localStorage.setItem('openedSidePanel', this.model);
```

suggestion: aligning with current naming convention for local storage keys

<img width="188" height="119" alt="image" src="https://github.com/user-attachments/assets/7e0d3204-02f4-45e4-a49f-37edceae33f4" />